lib/livebook_web/live/session_live/secrets_component.ex

```diff
@@ -22,61 +22,85 @@ defmodule LivebookWeb.SessionLive.SecretsComponent do
       <h3 class="text-2xl font-semibold text-gray-800">
         Add secret
```

Comment:
Let's make it "Select secret" when in select mode :)

Reply:
@jonatanklosko we are going to allow Kino to set the title, because for `kino_db`, a title such as "Set password" is best. So this is going to change soon anyway. :)

Reply:
Actually, we will use the options we have already left on the Kino side to rename it accordingly and with a fallback. But that is in my next PR :)

Reply:
Sounds good!